packages/playground/cli/src/cli.ts

```diff
@@ -296,6 +295,13 @@ async function run() {
 				? readAsFile(preinstalledWpContentPath)
 				: fetchWordPress(wpDetails.url, monitor);
 
+			const constants: Record<string, string | number | boolean | null> =
+				{
+					WP_DEBUG: args.debug,
+					WP_DEBUG_LOG: args.debug,
+					WP_DEBUG_DISPLAY: false,
```

Comment:
In your experience, is this something that most people prefer to disable? It's not clear to me that I'd want it disabled if running with the `--debug` flag because having messages printing in-page might be the feedback that is easiest to access.

Reply:
`WP_DEBUG_DISPLAY` is rarely enabled. It literally prints all error levels into the HTML and most of the time messes up the page. 

But not when I think of it, the point of the debug flag is to control the displaying of logs and collection.

I will set these the same way we do in the Website and users can always override them using blueprints.

Reply:
This is the debug flag description. It shouldn't control logging. 

https://github.com/WordPress/wordpress-playground/blob/3c95e981964b0190536301f45cefd2160d172849/packages/playground/cli/src/cli.ts#L107

Reply:
What does "return" mean in this context? It's not a function return

Reply:
And let's update the description to clarify this – is the `--debug` flag only effective at build time? Or does it also affect the built site later on?

Reply:
Ok! 

> is the --debug flag only effective at build time? Or does it also affect the runtime or the built site later on?

Only during the build, but it would be nice for it to also return errors during runtime.

Reply:
By "return" do you mean "print"? Or "save to file"? Or something else?

Reply:
It's printing, thanks for the questions, they help narrow down what's missing in the text.
Here is an improved version https://github.com/WordPress/wordpress-playground/pull/1985/files#diff-ff00c9c03a9587a4ac648349850e14a53fdd62418bce04573537c9c794c95811R107